tests/test_zonal_average.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import xarray as xr
+
+from pop_tools import DATASETS
+from pop_tools.regrid import zonal_average
+
+test_ds = xr.open_dataset(DATASETS.fetch('cesm_pop_monthly.T62_g17.nc'), chunks={})
+
+
+@pytest.mark.parametrize(
+    'ds, grid_name, dest_grid_method',
+    [(test_ds, 'POP_gx1v6', 'regular_grid'), (test_ds, 'POP_gx1v6', 'lat_axis')],
+)
+def test_zonal_average(ds, grid_name, dest_grid_method):
+    out = zonal_average(ds, grid_name, dest_grid_method)
+    assert isinstance(out, xr.Dataset)
+    assert 'region' in out.dims
+    assert 'TEMP' in out.variables
+    assert 14 == len(out.region)
```

Comment:
Could we produce some output with keith's za functionality, and then do some numerical comparison as part of the test?

Reply:
I am planning on holding off on this until we can figure out why it's next lining up exactly... It has something to do with the region masks, but I can't nail it down